src/Bybit/Spot/API/OrderBook.jl

```diff
@@ -0,0 +1,92 @@
+module OrderBook
+
+export OrderBookQuery,
+    OrderBookData,
+    order_book
+
+using Serde
+using Dates, NanoDates, TimeZones
+
+using CryptoAPIs.Bybit
+using CryptoAPIs.Bybit: Data, List, Rows
+using CryptoAPIs: Maybe, APIsRequest
+
+Base.@kwdef struct OrderBookQuery <: BybitPublicQuery
+    symbol::String
+    limit::Maybe{Int64} = nothing
+end
+
+struct Level <: BybitData
+    price::Float64
+    size::Float64
+end
+
+struct OrderBookData <: BybitData
+    asks::Vector{Level}
+    bids::Vector{Level}
+    time::NanoDate
+end
+
+"""
+    order_book(client::BybitClient, query::OrderBookQuery)
+    order_book(client::BybitClient = Bybit.Spot.public_client; kw...)
+
+[`GET /spot/v3/public/quote/depth`](https://bybit-exchange.github.io/docs/spot/public/depth)
+
+## Parameters:
+
+| Parameter | Type   | Required | Description |
+|:----------|:-------|:---------|:------------|
+| symbol    | String | true     |             |
+| limit     | Int64  | false    |             |
+
+## Code samples:
+
+```julia
+using Serde
+using CryptoAPIs.Bybit
+
+result = Bybit.Spot.order_book(
+    symbol = "ADAUSDT",
+)
```

Comment:
To continue about separating Args by `;`
```suggestion
result = Bybit.Spot.order_book(;
    symbol = "ADAUSDT",
)
```